curriculum/challenges/english/25-front-end-development/quiz-javascript-dates/66edd3711bb9f7efa73aef91.md

```diff
@@ -17,439 +17,455 @@ Answer all of the questions below correctly to pass the quiz.
 
-Placeholder answer
+`toString()`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What would output `console.log(new Date().getFullYear());`, if it would be June 12th 2022?
 
 #### --distractors--
 
-Placeholder distractor 1
+`"June 12th 2022"`
 
 ---
 
-Placeholder distractor 2
+`2023`
 
 ---
 
-Placeholder distractor 3
+`06/12/2022`
 
 #### --answer--
 
-Placeholder answer
+`2022`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What will be the output of the following code?
+
+```js
+console.log(new Date("2023-06-18T12:00:00").toUTCString());
+```
 
 #### --distractors--
 
-Placeholder distractor 1
+`Mon, 19 Jun 2023 12:00:00 GMT`
 
 ---
 
-Placeholder distractor 2
+`Sun, 18 Jun 2023 12:00:00 GMT+1`
 
 ---
 
-Placeholder distractor 3
+`Sun, 18 Jun 2023 13:00:00 GMT`
 
 #### --answer--
 
-Placeholder answer
+`Sun, 18 Jun 2023 12:00:00 GMT`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+If the time in your locale is formatted as `HH:MM:SS AM/PM`, which line of code would correctly display the current time?
 
 #### --distractors--
 
-Placeholder distractor 1
+`console.log(new Date().toLocaleDateString());`
 
 ---
 
-Placeholder distractor 2
+`console.log(new Date().toString());`
 
 ---
 
-Placeholder distractor 3
+`console.log(new Date().toCityDateString());`
 
 #### --answer--
 
-Placeholder answer
+`console.log(new Date().toLocaleTimeString());`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+The `toISOString()` method formats a date as?
 
 #### --distractors--
 
-Placeholder distractor 1
+A USA-863 string format.
 
 ---
 
-Placeholder distractor 2
+An MP3 string format.
 
 ---
 
-Placeholder distractor 3
+A UTC string format.
 
 #### --answer--
 
-Placeholder answer
+An ISO-8601 string format.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What should an ISO-8601 date format look like?
 
 #### --distractors--
 
-Placeholder distractor 1
+DD-MM-YYYY
 
 ---
 
-Placeholder distractor 2
+DD-MM-YYTHH
 
 ---
 
-Placeholder distractor 3
+YYYY-MM-DDTHH:mm:ssZ
 
 #### --answer--
 
-Placeholder answer
+YYYY-MM-DDTHH:mm:ss.sssZ
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the output of `console.log(new Date(2003, 6, 27).getMonth());`?
 
 #### --distractors--
 
-Placeholder distractor 1
+`"27/6/2003"`
 
 ---
 
-Placeholder distractor 2
+`27`
 
 ---
 
-Placeholder distractor 3
+`2003`
 
 #### --answer--
 
-Placeholder answer
+`6`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+How would you format a date to a locale-specific string or a more readable format?
 
 #### --distractors--
 
-Placeholder distractor 1
+`.toLocaleDate()`
 
 ---
 
-Placeholder distractor 2
+`.toLocaleString()`
 
 ---
 
-Placeholder distractor 3
+`.toLocaleTimetring()`
 
 #### --answer--
 
-Placeholder answer
+`.toLocaleDateString()`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which option will correctly create a `Date` object representing January 1, 2025?
```

Comment:
```
new Date("1-1-2025")
Wed Jan 01 2025 00:00:00 GMT+0100
new Date(2025, 0, 0);
Tue Dec 31 2024 00:00:00 GMT+0100
new Date("2025-01-01T00:00:00");
Wed Jan 01 2025 00:00:00 GMT+0100
new Date("2025-01-01")
Wed Jan 01 2025 01:00:00 GMT+0100
```
This isn't wrong (remember about local time-zone), but I worry it can be very confusing.

Reply:
I see what you mean by confusing, what if we format the question to this, what do u think ? This would remove the problem or issue we keep having with the users different time zone.

### --question--

#### --text--

Which option will correctly create a string representing January 1, 2025?

```js
const d = ?;
console.log(d.toISOString().split("T")[0]);
```

#### --distractors--

`const d = new Date("2021-02-01");`

---

`const d = new Date("2003-06-06");`

---

`const d = new Date("2001-04-08");`

#### --answer--

`const d = new Date("2025-01-01");`